CMakeLists.txt

```diff
@@ -147,6 +142,23 @@ endif()
 find_package(SDL2_net)
 
 # FluidSynth backend
+if (NOT FluidSynth_DIR)
+    if (WIN32 AND MSVC AND CMAKE_SIZEOF_VOID_P STREQUAL 4)
```

Comment:
How about cross building?

Reply:
Fluidsynth releases two different archives for 64-bit and 32-bit versions. We need to update these downloads from time to time, so I thought less is better.